os/fs/smartfs/smartfs.h

```diff
@@ -175,7 +175,7 @@
 #define INODE_STATE_FILE          (CONFIG_NXFFS_ERASEDSTATE ^ 0x22)
 #define INODE_STATE_DELETED       (CONFIG_NXFFS_ERASEDSTATE ^ 0xaa)
 
-/* Smartfs worbuffer maxuimum length */
+/* Smartfs worbuffer maximum length */
```

Comment:
is it supposed to be workbuffer instead of worbuffer?

Reply:
Agreed and updated,
Thanks